cypress/global.d.ts

```diff
@@ -33,6 +33,7 @@ declare global {
     // eslint-disable-next-line @typescript-eslint/no-namespace
     namespace Cypress {
         interface ApplicationWindow {
+            mxSettingsStore: any; // XXX: Importing SettingsStore causes a bunch of type lint errors
```

Comment:
like what? We shouldn't be using any - we should be fixing the errors.

Reply:
Huh, they seem to be gone now; no idea what happened

Reply:
Argh, hope, they aren't gone

Reply:
https://github.com/matrix-org/matrix-react-sdk/runs/7134469894?check_suite_focus=true is failing and I don't think I see why...